shell/platform/tizen/public/flutter_tizen.h

```diff
@@ -91,12 +91,10 @@ FlutterDesktopGetPluginRegistrar(FlutterDesktopEngineRef engine,
 FLUTTER_EXPORT FlutterDesktopMessengerRef
 FlutterDesktopEngineGetMessenger(FlutterDesktopEngineRef engine);
 
-/**
- * Return backing window for manipulation in host application.
- *
- * If run engine on a wearable device, need cast void* to Evas_Objct*,
- * otherwise cast void* to Ecore_Wl2_Window*.
- */
+// Returns a window for manipulation in host application.
```

Comment:
Since the input parameter is `registrar`, how about
```suggestion
// Returns the window associated with this registrar's engine instance.
```

Reply:
Much clearer, thanks!